src/core/network.js

```diff
@@ -441,6 +699,8 @@ var NetworkManager = (function NetworkManagerClosure() {
 
       var networkManager = this._manager;
       var fullRequestXhrId = this._fullRequestId;
+      var headers = this._parseHeaders();
```

Comment:
`this.filename = this._getFilenameFromHeaders(); // TODO: implement this method`